src/client/cashayPersistTransform.js

```diff
@@ -0,0 +1,24 @@
+import {createTransform} from 'redux-persist';
+import jwtDecode from 'jwt-decode';
+
+const cashayDeserializer = outboundState => {
+  const auth = outboundState.data.result.getUserWithAuthToken;
+  if (auth) {
+    const authObj = auth[''];
+    const {authToken} = authObj;
+    if (authToken) {
+      const authTokenObj = jwtDecode(authToken);
+      if (authTokenObj.exp < Date.now() / 1000) {
+        authObj.authToken = null;
```

Comment:
our entire store is persisted in `localStorage`. When we rehydrate it, we make sure the auth token is still valid so you're guaranteed to start your system with no auth token or a legit one.